bundles/org.openhab.binding.ipcamera/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -7,6 +7,7 @@
 	<thing-type id="group">
 		<label>Group Multiple Cameras</label>
 		<description>Show multiple cameras as a single rotating feed</description>
+		<semantic-equipment-tag>Display</semantic-equipment-tag>
```

Comment:
Not sure if this is right. A `Display` (physicall device) is something different then a feed (stream of bits)

Display has no synonyms or description in the https://github.com/openhab/openhab-core/blob/main/bundles/org.openhab.core.semantics/model/SemanticTags.csv file

Reply:
Ok. Removing..